experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/README.md

```diff
@@ -0,0 +1,10 @@
+|-----------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Sequential workflow   | [Sequential workflow](https://docs.langchain4j.dev/tutorials/agents/#sequential-workflow) | [LC4JEquivalenceIT.java#L43](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L43)                                              |
+| Loop workflow         | [Loop workflow](https://docs.langchain4j.dev/tutorials/agents/#loop-workflow)             | [LC4JEquivalenceIT.java#L76](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L76)<br>[LC4JEquivalenceIT.java#L112](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L112) |
+| Parallel workflow     | [Parallel workflow](https://docs.langchain4j.dev/tutorials/agents/#parallel-workflow)     | [LC4JEquivalenceIT.java#L154](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L154)                                           |
+| Conditional workflow  | [Conditional workflow](https://docs.langchain4j.dev/tutorials/agents/#conditional-workflow) | [LC4JEquivalenceIT.java#L218](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L218)                                           |
+| Error handling        | [Error handling](https://docs.langchain4j.dev/tutorials/agents/#error-handling)           | [LC4JEquivalenceIT.java#L185](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L185)                                           |
+| Human-in-the-loop     | [Human-in-the-loop](https://docs.langchain4j.dev/tutorials/agents/#human-in-the-loop)     | [LC4JEquivalenceIT.java#L257](https://github.com/treblereel/serverless-workflow-sdk-java/blob/langchain_examples/experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java#L257)                                           |
```

Comment:
Instead of this table, can you add the code excerpt? So the reader doesn't need to click on each link and go back and forth.